src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -85,14 +76,16 @@
 
 	protected KubernetesDeployerProperties properties;
 
+	protected DeploymentPropertiesResolver deploymentPropertiesResolver;
+
 	/**
 	 * Create the RuntimeEnvironmentInfo.
 	 *
 	 * @param spiClass the SPI interface class
 	 * @param implementationClass the SPI implementation class
 	 * @return the Kubernetes runtime environment info
 	 */
-	protected RuntimeEnvironmentInfo createRuntimeEnvironmentInfo(Class spiClass, Class implementationClass) {
```

Comment:
This method can be left as protected.   Seems like it is only used in child classes.